charts/rancher-csp-adapter/Chart.yaml

```diff
@@ -12,5 +12,5 @@ annotations:
   catalog.cattle.io/scope: management
   catalog.cattle.io/os: linux
   catalog.cattle.io/permits-os: linux
-  catalog.cattle.io/rancher-version: '>= 2.6.6-0 < 2.7.0-0'
+  catalog.cattle.io/rancher-version: '>= 2.7.0-0 < 2.8.0-0'
   catalog.cattle.io/kube-version: '>= 1.22.0-0 < 1.25.0-0'

```

Comment:
Can you fix the kube-version as well while you are at it?

Reply:
IIRC this version is more for the UI than anything else, and since the UI is not used when installing this the value isn't super important.

That being said, I think we should make this accurate. This doesn't support 1.24 right now, and we are going to need to fix that. Hiding that fact by not limiting the version doesn't help anyone.

In short, we should set to `<1.24` now and open an issue to get it to work with 1.24. Then when someone makes the updates to get this to work with 1.24, they can update the chart accordingly.

Reply:
Agreed, will add the constraint. Question: do we know for sure if this change in 1.24 breaks the adapter's behavior?
https://itnext.io/big-change-in-k8s-1-24-about-serviceaccounts-and-their-secrets-4b909a4af4e0

I'll create an issue anyways, whether it's for investigation of whether this breaks anything, or for a definite fix.

Reply:
That's a good point. I think we can be reasonably sure that this does break the behavior (the adapter uses the service account token to call back to rancher for some node counts), but you are correct that we should confirm 100% before setting version constraints. 

We won't be able to confirm this until aws releases eks 1.24 or we get some docs on using this outside of EKS. Let's re-evaluate at that time.